src/components/TodoList/TodoList.tsx

```diff
@@ -1,12 +1,34 @@
 /* eslint-disable */
-import React from 'react';
+import React, { useEffect, useState } from 'react';
+import { Todo } from '../../types/Todo';
+import cn from 'classnames';
+import { useAppDispatch, useAppSelector } from '../../app/hooks';
+import { currentTodoSlice } from '../../features/currentTodo';
+import { getFilteredTodos } from '../../utils/getFilteredTodos';
 
 export const TodoList: React.FC = () => {
+  const todos = useAppSelector(state => state.todos);
+  const currentTodo = useAppSelector(state => state.currentTodo);
+  const filter = useAppSelector(state => state.filter)
```

Comment:
Missing semicolon at the end of this line. While this does not break the code, it's good practice to maintain consistency in code style.